src/common/options.cc

```diff
@@ -3259,6 +3259,11 @@ std::vector<Option> get_global_options() {
     .set_default(5)
     .set_description("Set the maximum number of times we will preempt a deep scrub due to a client operation before blocking client IO to complete the scrub"),
 
+    Option("osd_scrub_max_preempt_divisor", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
I'd really like for us to not introduce new config options unless it is unavoidable. I don't think users should have to set this.